clouddriver-appengine/src/main/java/com/netflix/spinnaker/clouddriver/appengine/storage/controllers/AppengineStorageController.java

```diff
@@ -28,15 +28,17 @@
 import java.util.ArrayList;
 import java.util.List;
 
+// TODO(jacobkiefer): Refactor this Controller into a common controller with injected StorageService(s) when we
+// add another storage account service. Leaving this in Appengine's scope for now.
 @Slf4j
 @RestController
-@RequestMapping("/appengine/storage")
+@RequestMapping("/storage")
 class AppengineStorageController {
 
   @Autowired(required = false)
-  StorageConfigurationProperties storageAccountInfo;
+  private StorageConfigurationProperties storageAccountInfo;
 
-  @RequestMapping(value = "/list", method = RequestMethod.GET)
+  @RequestMapping(method = RequestMethod.GET)
```

Comment:
I might call this `accounts` in case we need to get other things from this controller in the future

Reply:
Let's move that in a future refactor when we add another accounts source.

Reply:
I think the endpoints should be static, each time we change them we break older versions of other services. I'm fine if we move this class later, but we should try to keep the endpoints identical